force-app/main/default/Objects/fields/Token_Migration_Test__c.field-meta.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@jupadhyaya 
Please move to the correct folder!
You can check the folder in Fonteva Payments

Reply:
updated